bgpd/bgp_route.c

```diff
@@ -5347,11 +5347,10 @@ void bgp_update(struct peer *peer, const struct prefix *p, uint32_t addpath_id,
 			    peer->sort == BGP_PEER_EBGP &&
 			    CHECK_FLAG(pi->flags, BGP_PATH_HISTORY)) {
 				if (unlikely(bgp_debug_update(peer, p, NULL, 1))) {
-					bgp_debug_rdpfxpath2str(
-						afi, safi, prd, p, label,
-						num_labels, addpath_id ? 1 : 0,
-						addpath_id, evpn, pfx_buf,
-						sizeof(pfx_buf));
+					bgp_debug_rdpfxpath2str(afi, safi, prd, p, label,
+								num_labels, addpath_id ? 1 : 0,
+								addpath_id, attr_new->evpn_overlay,
```

Comment:
Curious what is the backtrace, because if `evpn` is NULL, it should be handled correctly by bgp_debug_rdpfxpath2str() unless I missed something.